controllers/app/main.go

```diff
@@ -71,7 +67,7 @@ func main() {
 		Port:                   9443,
 		HealthProbeBindAddress: probeAddr,
 		LeaderElection:         enableLeaderElection,
-		LeaderElectionID:       "5d8f3b44.sealos.io",
+		LeaderElectionID:       "f3e714c5.sealos.io",
```

Comment:
I Think we can generate controller code for app, to prevent future need to use

Reply:
Done